Dockerfile

```diff
@@ -64,3 +64,43 @@ COPY --from=secrets-provider-builder /opt/secrets-provider-for-k8s/secrets-provi
 
 ENTRYPOINT [ "/usr/local/bin/secrets-provider"]
 
+# =================== MAIN CONTAINER (REDHAT) ===================
+FROM registry.access.redhat.com/rhel as secrets-provider-for-k8s-redhat
+MAINTAINER CyberArk Software, Ltd.
+
+ARG VERSION
+
+LABEL name="secrets-provider-for-k8s"
+LABEL vendor="CyberArk"
+LABEL version="$VERSION"
+LABEL release="$VERSION"
+LABEL summary="Store secrets in Conjur or DAP and consume them in your Openshift application containers"
+LABEL description="The Secrets Provider for Kubernetes runs as an init container and fetches \
```

Comment:
The CyberArk Secrets Provider for Kubernetes runs as an init container and populates the Kubernetes secrets required by the pod, from Conjur.

This is from the doc. If you think it's not good please change it also there.

Reply:
updating README

Reply:
can you update this as well?